.github/workflows/ci.yaml

```diff
@@ -0,0 +1,55 @@
+name: kubectl plugin test
+on:
+  pull_request:
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  Plugin-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.4.2
+      with:
+        minikube version: 'v1.21.0'
+        kubernetes version: 'v1.19.2'
+        start args: --memory 6g --cpus=2
+        github token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: print k8s cluster status
+      run:  |
+        minikube status
+        kubectl get nodes
+
+    - name: use local disk
+      run: tests/github-action-helper.sh use_local_disk
+
+    - name: deploy rook cluster
+      run: tests/github-action-helper.sh deploy_rook
+
+    - name: wait for operator pod to be ready
+      run: tests/github-action-helper.sh wait_for_pod_to_be_ready_state
+
+    - name: Test pluging with ceph commands
+      run: |
+        # when we have the kubectl krew plugin available we can use like `kubectl rook-ceph` but for now in local deployment we have to use `kubectl rook` as kubectl plugin has this limitation.
+        # Doc link to clear the comment above:
+          # https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/
+          # https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#naming-a-plugin
+          # https://krew.sigs.k8s.io/docs/developer-guide/distributing-with-krew/
+        sudo mv kubectl-rook-ceph.sh /usr/local/bin/kubectl-rook
```

Comment:
```suggestion
        sudo mv kubectl-rook-ceph.sh /usr/local/bin/kubectl-rook-ceph
```

Reply:
this is temporary for deployment. please read the above comment in the file.

Reply:
So the CI needs to install krew? The CI should use it exactly as we recommend it to users.